src/CropTypeListBuilder.php

```diff
@@ -82,9 +83,25 @@ public function buildRow(EntityInterface $entity) {
     ];
     $row['description'] = Xss::filterAdmin($entity->description);
     $row['aspect_ratio'] = $entity->getAspectRatio();
+
+    // Load all image styles where the current crop type is used.
+    $usage = array();
+    $image_style_ids = \Drupal::entityQuery('image_style')->condition('effects.*.data.crop_type', $entity->id())->execute();
+    $image_styles = entity_load_multiple('image_style', $image_style_ids);
```

Comment:
This function is deprecated. Let's ImageStyle::loadMultiple() or inject storage.